app/src/components/TokenCard.tsx

```diff
@@ -0,0 +1,23 @@
+
+import React from 'react';
+
+import { Box, Direction, Image, PaddingSize, Spacing, Stack, Text, TextAlignment } from '@kibalabs/ui-react';
+
+export const TokenCard = (): React.ReactElement => {
+  return (
+    <Stack direction={Direction.Vertical} isFullHeight={true} isFullWidth={true} padding={PaddingSize.Wide}>
+      <Text variant='header3-singleLine'>Your Holdings</Text>
+      <Spacing variant={PaddingSize.Wide} />
+      <Box variant='card' height='204px' width='140px'>
+        <Box height='140px' width='140px'>
+          <Image source={'/assets/icon.png'} alternativeText='image' fitType='contain' />
+        </Box>
+        <Spacing variant={PaddingSize.Wide} />
+        <Text variant='subtitle' alignment={TextAlignment.Center}>mdtp</Text>
```

Comment:
these fonts seems badly chosen - if its taking a while to decide just use the default and leave a comment on the PR for me to choose